bump_rpm.sh

```diff
@@ -64,13 +64,19 @@ if [[ $CURRENT_VERSION != $NEW_VERSION ]] ; then
 	git annex add *.gem
 	git add $SPEC_FILE
 
-	echo "TODO:"
 	if [[ $PACKAGE_NAME == rubygem-* ]] ; then
```

Comment:
should this also match `tfm-rubygem-*`?

Reply:
There is one case where we have expanded the package name in the directory name so yes.